src/ui/public/stringify/types/Color.js

```diff
@@ -0,0 +1,59 @@
+define(function (require) {
+  return function _StringProvider(Private) {
+    var _ = require('lodash');
+    var FieldFormat = Private(require('ui/index_patterns/_field_format/FieldFormat'));
+
+    require('ui/field_format_editor/samples/samples');
```

Comment:
This line isn't needed anymore since you've changed the way you're displaying samples.